docs/Examples/MultipartUpload.md

```diff
@@ -0,0 +1,57 @@
+# Multipart upload
+
+Suppose we want to upload a GIF with additional parameters in one request. How to approach this problem? It depends if our parameters should be in body of the request or in the URL.
+
+## Setup
+
+Let's say we have our `TargetType` already setup - it's called `MyService`:
+
+```swift
+public enum MyService {
+    case uploadGif(Data, description: String)
+}
+```
+
+Here, our additional parameter is `description`, which is a `String`.
+
+## Parameters in body
+
+When we want to add parameters to the body of our request, we need to append them as a `MultipartFormData` in the `task` property:
```

Comment:
I would change to something like
> When we want to perform a multipart request with a request body, we have to override the `task` property,  create a `MultiPartFormatData` for each of our parts and them return a ` .uploadMultiPart` task with the appended parts.

Reply:
We probably shouldn't use the word `override`

Reply:
Exactly, but few good improvements there so I've added them as well!